common/src/main/java/com/wynntils/commands/LootrunCommand.java

```diff
@@ -0,0 +1,427 @@
+    }
+
+    private int removeChest(CommandContext<CommandSourceStack> context) throws CommandSyntaxException {
+        BlockPos pos = BlockPosArgument.getSpawnablePos(context, "pos");
+
+        LootrunUncompiled current = LootrunUtils.getActiveLootrun();
+
+        if (current == null) return 0;
+
+        if (current.chests().remove(pos)) {
+            context.getSource()
+                    .sendSuccess(
+                            new TranslatableComponent("feature.wynntils.lootrunUtils.chestRemoved", pos.toShortString())
+                                    .withStyle(ChatFormatting.GREEN),
+                            false);
+        } else {
+            context.getSource()
+                    .sendFailure(new TranslatableComponent(
+                            "feature.wynntils.lootrunUtils.chestDoesNotExist", pos.toShortString()));
+        }
+
+        return LootrunUtils.getEditLootrun(context.getSource(), true);
+    }
+
+    private int undoLootrun(CommandContext<CommandSourceStack> context) {
+        if (LootrunUtils.recording == null) {
+            context.getSource().sendFailure(new TranslatableComponent("feature.wynntils.lootrunUtils.notRecording"));
+            return 0;
+        } else {
+            Vec3 position = McUtils.player().position();
+            List<Vec3> points = LootrunUtils.recording.points();
+            List<Vec3> removed = new ArrayList<>();
+            boolean left = false;
+            for (int i = points.size() - 1; i >= 0; i--) {
+                if (i == 0) {
+                    if (left) {
+                        context.getSource()
+                                .sendFailure(new TranslatableComponent("feature.wynntils.lootrunUtils.undoStandNear"));
+                    } else {
+                        context.getSource()
+                                .sendFailure(
+                                        new TranslatableComponent("feature.wynntils.lootrunUtils.undoNotFarEnough"));
+                    }
+                    return 0;
+                }
+
+                if (points.get(i).distanceToSqr(position) < 4) {
+                    if (left) {
+                        break;
+                    }
+                } else {
+                    left = true;
+                }
+
+                removed.add(points.get(i));
+            }
+
+            points.removeAll(removed);
+            context.getSource()
+                    .sendSuccess(new TranslatableComponent("feature.wynntils.lootrunUtils.undoSuccessful"), false);
+            LootrunUtils.recordingInformation.setDirty(true);
+            return removed.size();
+        }
+    }
+
+    private int folderLootrun(CommandContext<CommandSourceStack> context) {
+        Util.getPlatform().openFile(LootrunUtils.LOOTRUNS);
+        return 1;
+    }
+
+    @Override
+    public void register(CommandDispatcher<CommandSourceStack> dispatcher) {
+        dispatcher.register(Commands.literal("lootrun")
```

Comment:
Could you add an alias to "lr" like Wynntils 1.12.2? I can do it if you don't want to